onnx_tf/handlers/backend/depth_to_space.py

```diff
@@ -40,7 +40,8 @@ def _common(cls, node, **kwargs):
     if mode == "CRD":
       # need native computation
       bsize = attrs.get("blocksize")
-      batch, channel, height, width = x.shape
+      x_shape = tf.shape(x)
```

Comment:
First, thanks very much for making the contribution! If you don't mind, please consider use the utility method tf_shape, which takes care of both fully defined and dynamic shapes, and provides slight optimization for static shape.

Reply:
Thanks for the suggestion, switched to tf_shape.